src/contrib/mlir/lowerer.cpp

```diff
@@ -526,6 +526,58 @@ namespace
         return matchSuccess();
     }
 
```

Comment:
You can add a utility function to lower all the Unary Op's , look at `lower_binary_elementwise` for example

Reply:
Thats for another PR